palantirnet/st-bernards
# MSSB-37: Module updates

## JIRA Ticket(s)

- See ticket [MSSB-37: Update contrib modules](https://palantir.atlassian.net/browse/MSSB-37).

## Description

- Downgrades Contribute to the 1.x beta.
- Updates Fancy File Delete to the latest dev.
- Update CKEditor iFrame to the stable release.
- Updates all other modules (except Media Entity and Video Embed Field) to the recommended releases.
- Renames `robots.txt`, so it can be managed by the Robotstxt module. (This file was mistakenly added when updating the site for our toolchain.)

## Testing instructions

- `composer install`
- `phing build`
- Import a production database.
- `drush cr`
- `drush updb`
- `drush cr`
- `drush cim -y`
- `drush uublk 1`
- `drush uli`
- Visit the "available updates" admin page, and notice all the green under "Modules". (Note: Fancy File Delete is using the latest dev version but will still show "2015-Oct-10" due to an issue with packages.drupal.org; https://www.drupal.org/project/project_composer/issues/2994170.)
- Verify the site functions as expected.
